build/config/ui.gni

```diff
@@ -20,14 +20,9 @@
 import("//build/config/chromecast_build.gni")
 import("//build/config/chromeos/args.gni")
 import("//build/config/chromeos/ui_mode.gni")
+import("//build/config/ozone.gni")
 
 declare_args() {
-  # Indicates if Ozone is enabled. Ozone is a low-level library layer for Linux
-  # that does not require X11.
-  use_ozone =
-      is_chromeos || (is_chromecast && !is_android) || is_fuchsia ||
-      is_linux && !is_starboard
-
   # Indicates if the UI toolkit depends on X11.
```

Comment:
Lines 26–30 aren't in the source revision:
https://chromium.googlesource.com/chromium/src/+/114.0.5735.331/build/config/ui.gni#25